.gitignore

```diff
@@ -25,3 +25,5 @@ phpunit.phar
 /vendor
 /composer.lock
 /node_modules
+
+package.json
```

Comment:
Why?
```suggestion
package-lock.json
```

Reply:
Scrutinizer generates an error with it. Temporary disabled. This file is regenerated every time on the composer update.